ckanext/canada/templates/internal/package/new_resource_not_draft.html

```diff
@@ -1,6 +1,5 @@
 {% ckan_extends %}
 
-{% block subtitle %}
+{% block secondary_content %}
+ {% snippet 'package/snippets/resources.html', pkg=pkg %}
```

Comment:
I think this one-line addition is worth sending upstream too. https://github.com/ckan/ckan/blob/f8865626012770a2e67d4caf93cae7dbd1e4f094/ckan/templates/package/new_resource_not_draft.html#L20

Reply:
done here https://github.com/ckan/ckan/pull/7889